.changeset/slimy-bananas-remain.md

```diff
@@ -0,0 +1,6 @@
+---
+'gqless': patch
```

Comment:
only need to release schema, can remove this line

Reply:
Fixed